client/src/routes/fiches/[id]/edit@blank.svelte

```diff
@@ -29,6 +37,7 @@
   import DatasetFormLayout from "src/lib/components/DatasetFormLayout/DatasetFormLayout.svelte";
 
   export let dataset: Maybe<Dataset>;
+  export let tags: Tag[];
```

Comment:
Vu l'utilisation de `Maybe.Some(tags)` et le résultat que devrait renvoyer `getTags()` (cf un autre commentaire), on devrait ici utiliser :

```suggestion
  export let tags: Maybe<Tag[]>;
```